### PR TITLE
[fix bug 1239466] Add page for Data Privacy Day.

### PR DESCRIPTION
**Pull request by jpetto**
@jbuck - Have a moment for code review? Hoping to get this up on the staging server early tomorrow (2/26) for stakeholders to give final review.

If not, I can ask one of the fine mozilla.org devs to take a look.

Thanks!

### CODE REVIEW

**Comment by jbuck** [comment]
@alicoding @ScottDowne too

File: lib/campaigns.js [hunk 1]
```diff
@@ -4,6 +4,12 @@ var campaigns = {
     lang: 'de',
     newsletters: 'mozilla-foundation',
     source_url: 'https://petitions.mozilla.org/data-retention/'
+  },
+  'stay-secure': {
+    bsd: '',
```
Comment (ScottDowne) [inline lib/campaigns.js]: I think my only question is: if this is a petition, how are you storing firstname and lastname? I didn't think basket was capable of that, if it is, I am interested in how as I would use it elswhere :)

**Comment by ScottDowne** [comment]
Just one comment, but I don't think it should block, otherwise it all looks pretty standard and I hold no strong opinions :)

@jbuck I'm fine with this going on staging.

File: lib/campaigns.js (hunk 1 above)
Comment (jpetto) [inline lib/campaigns.js]: Ah, good catch! Basket cannot store first/last name.

**Comment by jpetto** [comment]
Just pushed another commit with some minor-ish changes - biggest being the removal of first/last name fields. If we're good to go, I'll squash the commits.

After squash and merge, how long until the changes are on staging?

**Comment by jbuck** [comment]
@jpetto r+, can you squash?

**Comment by jpetto** [comment]
Squashed!

**Pull request merged**